tests/ssl-opt.sh

```diff
@@ -8846,8 +8846,6 @@ run_test    "DTLS reordering: Buffer out-of-order hs msg before reassembling nex
             hs_timeout=2500-60000" \
             0 \
             -c "Buffering HS message" \
-            -c "attempt to make space by freeing buffered future messages" \
```

Comment:
These lines are testing the presence of these messages in the client output. A couple of extra messages wouldn't make them fail.

Also, removing these lines changes the purpose of these tests. I think we should not change observable behaviour in this PR. If we keep the observable behaviour we won't need to change any tests.